defaults/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+
+sansible_vernemq:
+  version: ~
```

Comment:
Can you alphabetically order these vars?

Reply:
Fixed.